### PR TITLE
Feature develop

### PR DESCRIPTION
**Pull request by GransonO**
## What does this PR do?
This PR adds an api end point to the project

## Description of Task to be completed?
Develop branch with all features combined

## How should this be manually tested?
The project can be forked from the repo. Install all packages from the requirements.txt file and from your command line, run the run.py file to start the server. Using post man, test the endpoints

## What are the relevant pivotal tracker stories?
Pivotal tracker story ID :  [2198167](https://www.pivotaltracker.com/n/projects/2198167)

### CODE REVIEW

File: .pytest_cache/v/cache/nodeids [hunk 1]
```diff
@@ -0,0 +1 @@
+[]
```
Comment (mwaz) [inline .pytest_cache/v/cache/nodeids]: add the .pythest_cache directory to .gitignore

Comment (GransonO) [reply .pytest_cache/v/cache/nodeids]: It's been added as requested. Thanks for the feedback

File: server/app/.coverage [hunk 2]
```diff
@@ -0,0 +1 @@
....py: This is a private format, don't read it directly!{"lines":{"I:\\andela_bootcamp_32\\fast_food_fastv2\\server\\app\\test\\test_create_order.py":[1,2,3,4,5,6,7,8,9,10,14,16,17,18,19,20],"I:\\andela_bootcamp_32\\fast_food_fastv2\\server\\app\\test\\test_data_handler.py":[2,3,4,5,6,10,22,44,50,46,47,51,52,54,79,55,57,58,59,60,61,62,63,64,65,67,68,73,74,75,77,80,70,82,110,85,86,112,154,113,116,117,118,119,120,121,122,123,124,126,128,132,133,135,136,137,138,139,140,141,142,143,145,149,152,155,147,150,156,158,180,160,162,163,164,165,166,167,168,169,170,172,174,175,176,178,181,182,12,13,14,24,25,27,28,29,30,31,32,33,34,35,37,39,40,41,89,90,92,93,94,95,96,97,98,99,100,102,106,109],"I:\\andela_bootcamp_32\\fast_food_fastv2\\server\\app\\test\\test_delete_order.py":[1,4,13,6,7,8,9,10,11,15,16,17,18,19,20],"I:\\andela_bootcamp_32\\fast_food_fastv2\\server\\app\\test\\all_get\\test_get_empty.py":[2,7,4,5,9,10],"I:\\andela_bootcamp_32\\fast_food_fastv2\\server\\app\\test\\all_get\\test_get_present.py":[2,4,5,6,7,8,9,10,11,12,13,14,15],"I:\\andela_bootcamp_32\\fast_food_fastv2\\server\\app\\test\\all_get\\test_get_specific.py":[2,14,27,4,5,6,7,8,9,10,11,16,17,18,19,20,21,22,23,24,29,30,31]}}
```
Comment (mwaz) [inline server/app/.coverage:1]: you can add this file to .gitignore

File: .pytest_cache/README.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# pytest cache directory #
```
Comment (mwaz) [inline .pytest_cache/README.md]: Did you add the  .pytest_cache to .gitignore ?

If you already have, run the following command to remove it from the cache: `git rm --cached  .pytest_cache`

**Comment by GransonO** [comment]
For the heroku deployment, you have to change from Https to http as the swagger api fails to render in https.

File: server/app/.coverage (hunk 2 above)
Comment (GransonO) [reply server/app/.coverage:1]: Done as requested.

File: .pytest_cache/README.md (hunk 3 above)
Comment (GransonO) [reply .pytest_cache/README.md]: Done as requested.

**Pull request closed without merging**